rex/resource_extraction/wind_cli.py

```diff
@@ -110,10 +110,14 @@ def sam_datasets(ctx, hub_height, lat_lon, gid, sites):
         logger.info('Saving data to {}'.format(ctx.obj['OUT_DIR']))
         with ctx.obj['CLS'](ctx.obj['H5'], **ctx.obj['CLS_KWARGS']) as f:
             if lat_lon is not None:
+                out_path = os.path.join(ctx.obj['OUT_DIR'],
+                                        'SAM_{}.csv'.format(lat_lon))
                 f.get_SAM_lat_lon(hub_height, lat_lon,
-                                  out_path=ctx.obj['OUT_DIR'])
+                                  out_path=out_path)
```

Comment:
Was this a bug? OR does this need to be updated in the other CLIs?

Reply:
yeah you know im not 100% sure how that worked! I added an assert statement in the `get_SAM_lat_lon()` and `get_SAM_gid()` methods that checked the out path ended in .csv and it failed here!